### ISSUE TITLE
[solc 0.5.0] Support calldata

### ISSUE THREAD

**Issue by roderik** [duaraghav8/Ethlint#243]
adding calldata to the _metadata field is required by solc v0.5.0 but solium trips up
```
  function enlist(uint _stakeAmount, uint _price, string calldata _metadata) external {
    // Stake must be above a certain amount
    require(_stakeAmount >= minEnlistAmount, "_stakeAmount >= minEnlistAmount");

    // Transfers tokens from user to Registry contract
    require(token.transferFrom(msg.sender, address(this), _stakeAmount), "transfer failed");

    // Add listing to listings
    listingFactory.createListing(
      msg.sender,
      _price,
      _stakeAmount,
      address(this),
      _metadata
    );
  }
```

```
> @settlemint/solidity-mint@0.0.0-development lint:contracts /Users/roderik/Development/solidity-mint
> solium --dir contracts --fix


contracts/tokencuratedregistry/TokenCuratedRegistry.sol
  94:67    error    Syntax error: unexpected token _    

✖ 1 error found.
```

**Comment by robrobbins** [comment]
+1

**Comment by duaraghav8** [comment]
solved in `v1.2.0`

This linter is now being published simultaneously to npm packages `solium` and `ethlint`. Although not mandatory, I highly recommend that you move to `ethlint`.

See [changelog](https://github.com/duaraghav8/Ethlint/blob/master/CHANGELOG.md#120-2018-12-25-santa)
Please let me know in case any of the 0.5 features are still causing trouble.
